test/lib/mocked-dockerode.ts

```diff
@@ -15,6 +15,24 @@ export class NotFoundError extends TypedError {
 
 const overrides: Dictionary<(...args: any[]) => Resolvable<any>> = {};
 
+interface Action {
+	name: string;
+	parameters: Dictionary<any>;
+}
+
+export let actions: Action[] = [];
```

Comment:
I couldn't think of a good way to add a Spy on this so had to create this...if anyone knows a better way let me know.